src/commands/dev/index.ts

```diff
@@ -0,0 +1,46 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { Category } from 'types/applicationCommand';
+
+export const data = new SlashCommandBuilder()
+    .setName('dev')
+    .setDescription("Commands intended to ease in Fluorine's development.")
+    .addSubcommand(subcommand =>
+        subcommand
+            .setName('reload')
+            .setDescription('Reloads a module.')
+            .addStringOption(option =>
+                option
+                    .setName('type')
+                    .setDescription('The type of the module.')
+                    .setRequired(true)
+                    .setChoices([
+                        ['Event', 'events'],
+                        ['Chat Input Command', 'commands'],
+                        ['Context Menu Command', 'context'],
+                        ['Component', 'components']
+                    ])
+            )
+            .addStringOption(option =>
+                option.setName('module').setDescription("The module that you're reloading.").setRequired(true)
+            )
+    )
+    .addSubcommand(subcommand =>
+        subcommand
+            .setName('shell')
+            .setDescription('Execute a shell script')
+            .addStringOption(option => option.setName('script').setDescription('The shell script.').setRequired(true))
+    );
+
+if (process.env.NODE_ENV === 'development') {
+    data.addSubcommand(subcommand =>
+        subcommand
+            .setName('eval')
+            .setDescription('Evaluates a given exprssion.')
+            .addStringOption(option => option.setName('code').setDescription('The code to evaluate.').setRequired(true))
+    );
+} else {
+    data.addSubcommand(subcommand => subcommand.setName('update').setDescription('Update the bot.'));
+}
```

Comment:
do you really use it in development

Reply:
wont hurt to have

Reply:
i still don't see your point.

Reply:
i dont think anything would explode if we allow /update to be used in a dev enviroment

Reply:
i mean if we were to allow /dev update on dev it would probably be updated to allow merging origin/main in but thatd be bad since it could break on the dev branch when we implement it.

Reply:
also it could break since `git pull` on a branch just pulls on that branch and doesnt pull from main or other branches.